<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Third_page</class>
 <widget class="QDialog" name="Third_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Background_image">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>-2</y>
     <width>964</width>
     <height>544</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Background_image_qrc_file.qrc">
     <normaloff>:/BGI_sets/Second_bgi.jpg</normaloff>:/BGI_sets/Second_bgi.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>960</width>
     <height>540</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="show_text1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>960</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>------------------------------------------------------------------------------------------------------------------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_text2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>517</y>
     <width>480</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>에빙하우스의 망각곡선 이론을 활용한 어휘력 끝장내기 - 김건우</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_text3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>517</y>
     <width>440</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>대전교육정보원 부설영재교육원 개인주제탐구발표대회 제출작</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="enter_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>600</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check_button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>290</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="answer_true_false">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>445</y>
     <width>540</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>단어의 뜻을 입력해 주세요.</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_to_user">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>780</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>눈 위에 서리가 덮인다는 뜻으로, 난처한 일이나 불행한 일이 잇따라 일어남을 이르는 말.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="step_show">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>445</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Step : 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>0</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Background_image_qrc_file.qrc"/>
 </resources>
 <connections/>
</ui>
